src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    public  Random random = new Random();
```

Comment:
The 'random' field should be private and preferably 'final' if it's not meant to be changed after instantiation of 'ColorSupplier'. Exposing it as 'public' allows for unwanted external modifications.